appveyor.yml

```diff
@@ -1,9 +1,9 @@
 environment:
   matrix:
-#  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
-  - JULIAVERSION: "julialang/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
-#  - JULIAVERSION: "julianightlies/bin/winnt/x86/julia-latest-win32.exe"
```

Comment:
Whoops, sorry.  This was done by a script that didn't pay attention as closely as either of us would have liked.  I'll fix this in the PR.